src/elements/footer-social.html

```diff
@@ -0,0 +1,212 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../bower_components/paper-fab/paper-fab.html">
+
+
+<dom-module id="footer-social">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+    <style>
+      :host {
+        padding-left: 4px;
+        position: relative;
+        color: var(--footer-text-color);;
+      }
+
+      .title {
+        display: inline-block;
+        text-transform: uppercase;
+        font-weight: 500;
+        font-size: 14px;
+        margin: 0;
+      }
+
+      .nav-inline {
+        display: inline;
+        margin: 0 55px 0 4px;
+      }
+
+      ul.nav-inline {
+        padding-left: 10px;
+      }
+
+      .nav-inline li {
+        display: inline-block;
+      }
+
+      .share-gplus {
+        height: 30px;
+        padding: 8px;
+        width: 23px;
+      }
+
+      .share-gplus paper-icon-button {
+        color: var(--red-gplus-color);
+      }
+
+      a {
+        display: inline-block;
+        margin: 0;
+        color: var(--footer-text-color);
+        text-decoration: none;
+      }
+
+      .social-group {
+        margin-right: 30px;
+        box-sizing: border-box;
+      }
+
+      .social-group.email h4 {
+        padding-right: 0;
+        padding-top: 17px;
+      }
+
+      .social-group.email a {
+        border-bottom: 1px solid var(--footer-text-color);
+        padding-bottom: 1px;
+      }
+
+      .social-group.social-networks {
+        padding-top: 5px;
+      }
+
+      .social-group.social-networks ul {
+        list-style-type: disc;
+      }
+
+      .social-group.blog {
+        padding-top: 17px;
+      }
+
+      .blog h4 {
+        padding-right: 55px;
+      }
+
+      .blog a {
+        border-bottom: 1px solid var(--footer-text-color);
+        padding-bottom: 1px;
+      }
+
+      .fab paper-fab {
+        transform: scale(1);
+        background: #fff;
+        color: inherit;
+        transition: transform 200ms cubic-bezier(0, 0, 0.2, 1), -webkit-transform 200ms cubic-bezier(0, 0, 0.2, 1);
```

Comment:
A new animation, can we use the previous one?
Write in a canonical way, we are going to use autoprefixer then